examples/README.md

```diff
@@ -0,0 +1,35 @@
+# Examples
+
+Here we have some examples for you to know how to not only use
+`react-testing-library` but also in general how to test common scenarios that
+pop up with React. Check out the `__tests__` directory for the different
+examples.
+
+## Setup
+
+The examples have a unique jest/eslint set up so the test files will resemble
+how they might appear in your project. (You'll see in the tests that we can
+`import {render} from 'react-testing-library'`).
+
+Another thing that we do for the tests is we have a `setup-test-env.js` file in
```

Comment:
Could this be made a bit more obvious by putting it in `jest.setup.js` or something more similar to how Jest is typically configured?

Reply:
I've never heard of `jest.setup.js` before. I'm not sure how much more obvious that is. Though I did consider just adding `import 'jest-dom/extend-expect'` in every file that needs the extensions to make it more obvious. That's probably the best idea. I'll do that.

Reply:
Actually it's fine as-is. I think maybe long-term the examples should drop the dependency on kcd-scripts to show a more direct Jest config, but that can be done later.

Reply:
nvm about `jest.setup.js` - it was manually referenced in my jest config, not automatic.

Reply:
I think that we need to have an example repo (somewhere else) that shows a good setup. Maybe one that uses create-react-app as well. We could just link to those in the examples.